File: courses/csharp-introduction/age_groups.cs
using System;

public class Program
{
    static void Main(string[] args)
    {
        int age = Convert.ToInt32(Console.ReadLine());

        // your code goes here
        if (age >= 0 && age <= 11) {
            Console.WriteLine("Child");
        }
        else if (age >= 12 && age <= 17) {
            Console.WriteLine("Teen");
        }
        else if (age >= 18 && age <= 64) {
            Console.WriteLine("Adult");
        }
    }
}

File: courses/kotlin/Control Flow/Age Groups.kt
fun main(args: Array<String>) {
    var age = readLine()!!.toInt()

    if (age < 0)
        println("Invalid age")
    else if (age <= 11)
        println("Child")
    else if (age <= 17)
        println("Teen")
    else if (age <= 64)
        println("Adult")
    else
        println("Senior")
}